packages/zoe/tools/manualTimer.js

```diff
@@ -9,6 +9,7 @@ import { Far } from '@agoric/marshal';
 import './types.js';
 import './internal-types.js';
 import { makeNotifierKit } from '@agoric/notifier';
+import { makePromiseKit } from '@agoric/promise-kit';
```

Comment:
Can we drop `packages/swingset-runner/demo/zoeTests/manualTimer.js` and import this code instead?